src/app/main/component/events/components/events-list/events-list.component.html

```diff
@@ -21,8 +21,8 @@
     </div>
     <div class="filter-container">
       <p class="sort-by">{{ 'homepage.events.filter-title' | translate }}</p>
-      <div class="filter-list">
-        <div class="dropdown">
+      <div class="filter-list" role="Sort by">
+        <div class="dropdown" tabindex="0" role="Sort by time">
```

Comment:
Check out the list of roles here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles
If you want to use these titles as hidden headings - use aria-label or aria-labelledby if you have element with required text on page; and be sure to use i18n with them, not just text in english